.github/workflows/realease-on-pr-merge.yaml

```
@@ -65,4 +64,4 @@ jobs:
           name: Release ${{ steps.bump_version.outputs.new_tag }}
           body: |
             Changes in this Release:
-            ${{ github.event.pull_request.body }}
+            ${{ github.event.pull_request.body }}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)